src/validation.cpp

```diff
@@ -3136,7 +3151,9 @@ void static UpdateTip(const CBlockIndex *pindexNew, const CChainParams& chainPar
         // Check the version of the last 100 blocks to see if we need to upgrade:
         for (int i = 0; i < 100 && pindex != nullptr; i++)
         {
-            int32_t nExpectedVersion = ComputeBlockVersion(pindex->pprev, chainParams.GetConsensus());
+            // TODO, we might have wallets telling users to update because of the new Pow Algo bits in the version.
```

Comment:
Should this TODO be done?

Reply:
I wanted to check to see if the current QT wallet even shows this error. If it does, we can add the code to bypass certain bits in this version

Reply:
Yes, I have seen the message display in the QT wallet.

Reply:
Done